src/azul/service/hca_response_v5.py

```diff
@@ -353,6 +353,7 @@ def make_projects(self, entry):
                 for publication in translated_project['publications']:
                     for key in list(publication.keys()):
                         publication[to_camel_case(key)] = publication.pop(key)
+                translated_project['accessions'] = project.get('accessions', [None])
```

Comment:
Your changes should follow the existing accession handling code. Generally speaking, if in doubt, append changes.

Here and everywhere.